src/tests/dictionaryConcepts/components/CreateMapping.test.js

```diff
@@ -74,4 +74,18 @@ describe('Test suite for dictionary concepts components', () => {
     const inputs = wrapper.find('select');
     expect(inputs).toHaveLength(2);
   });
+  it('should set the state to an empty string whenever there is a change in the value of the props received', () => {
+    const newProps = {
+      ...props,
+      isNew: true,
+      source: 'Source',
+    };
+    wrapper = mount(<CreateMapping {...newProps} />);
+    const nextProps = {
+      ...props,
+      source: 'MapType',
+    };
+    wrapper.setProps(nextProps);
+    expect(wrapper.instance().props.source).toEqual('MapType');
```

Comment:
How is this related to "should set the state to an empty string...."?

Reply:
I have renamed it appropriately